sr_hand_config/4650/controls/host/common/friction_compensation.yaml

```diff
@@ -0,0 +1,4 @@
+sr_friction_map: [
+#["LFJ5", 1,  [[0.0, 0.0], [1.0, 0.0]]],
```

Comment:
IS there any reason to keep this commented out?

Reply:
That's how we have it across most if not all hands.

Reply:
And? Is there a reason why? Keeping commented out code is definitely not a good idea

Reply:
I don't think I am the right person to ask that. I have a few questions myself regarding the controller files (this config is for a hand lite yet, we have controllers defined for LF which I don't understand why). I am having a chat with Toni later, looking for some explanation and I am happy to add your question to the list. Might be worth having a look again at the structure and contents of the config files :)